Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Valeri Karpov <valkar207@gmail.com>",
            "Saghm Rossi <saghmrossi@gmail.com>",
            "Kevin Yeh <kevinyeah@utexas.edu>"]
 
-description = "An experimental MongoDB driver written by MongoDB interns."
+description = "A native-Rust MongoDB driver. Originally written by MongoDB interns, the driver is an unofficial but still actively maintained project."
```

Comment:
Hmmm, I was thinking it might be worth mentioning its origins because its the only Rust driver for MongoDB that's at all affiliated with the company. Maybe I should check to see if there's any recommended language that denotes that the driver is owned by MongoDB but not officially supported.